clients/admin-ui/src/features/common/nav/v2/NavTopBar.tsx

```diff
@@ -21,16 +21,21 @@ export const NavTopBar = () => {
       borderColor="gray.100"
     >
       {nav.groups.map((group) => {
-        // The group links to its first child's path.
-        const { path } = group.children[0]!;
+        // "Management" is navigated to via the gear icon, so don't display it in the nav
+        if (group.title !== "Management") {
```

Comment:
nit: I like to "return early" as opposed to nesting an if, i.e.

```typescript
if (group.title === "Management") {
  return null;
}

const { path } = group.children[0];
// etc.
```

fewer indents :)